tests/test_schemas.py

```diff
@@ -27,7 +27,6 @@ def test_library_schema_full(self):
     def test_library_schema_telemetry_conf_value(self):
         interfaces.library.assert_schema_point("/telemetry/proxy/api/v2/apmtelemetry", "$.payload.configuration[]")
 
```

Comment:
```suggestion

@bug(context.library < "python@v2.9.0.dev", reason="APPSEC-52845")
```


There is another occurence lower in this file, line 57